### DISCUSSION TITLE
New Daughter Board

### DISCUSSION THREAD

**Discussion by dave18** [xjtuecho/EBAZ4205#37] [Show and tell]
Hi

I've created a new daughter board for the EBAZ4205.  It offers 12 bit VGA, PS2 Keyboard and brings PS2 and other IO (excl VGA) to 2.54mm IO headers.  Everything is through hole for easy assembly/soldering.  I've updated the Wiki but here is the link to my repo https://github.com/dave18/EBAZ4205-Daughter-Board

I've got 8 spare prototype boards (same as the uploaded Gerber/Schematic but with some missing silkscreen), as PCBWay provide 10 and I only needed two, so if anyone wants one let me know and I'll post them out (free of charge to UK, cost of postage elsewhere), least I can do to give back to a community that helped me get up and running with the board.

First come first served.

Regards

Dave

**Comment by DavidJRichards** [comment]
Hi Dave, I'm not sure how I got the email about your new board or if in fact you'll get the reply I sent. Anyway I'd love one of your boards please, just the thing for my unexpanded other ebaz board. You may find that my multicomp conversion will run on it with very little modification. look here: https://github.com/DavidJRichards/EBAZ4205/tree/master/projects/multicomp-z80 
Kind regards, David.

**Reply by dave18** [reply]
Hi David

I expect you got notified due to the kind help you gave me fixing the corrupted NAND on my board. 

If you let me have your address I'll pop a board in the post, my email is fasterthanasnail (at) googlemail.com 

They are unpopulated but all the parts are easy to get hold of. The 2mm headers were hardest to find but I've put the part number on the Eagle board layout (the x changes depending whether you go for tin or gold plated leads). 

Cheers

Dave

**Comment by dave18** [comment]
I've found a stupid error in the board design.  I put in a 74LS245N instead of SN74LVC245AN.  The latter includes level shifting to take the PS2 keyboard inputs down from 5v for 3.3v.  I've taken the GIT private until I've tested whether dropping in the replacement works ok.

I also neglected to mention the I hacked by EBAZ4205 board to provide the 3.3v line on the Data Pins by removing the D21 and D22 diodes and running the 3.3v from J7 to the Data 1 and Data 2 VCC pins to provide VCC Out as I never plan on powering the board through the data pins.

**Reply by skiphansen** [reply]
Your board sounds very interesting, but the link is dead.  Is it coming back?  I was thinking along the same lines, but why reinvent the wheel?

**Reply by dave18** [reply]
@skiphansen 

I've updated the files and made the repository public again.  I've added to the notes the fact that I've modded my EBAZ4205 board to provide 3.3v power to the daughter board through the Data Pins.  I never need to power the board using these pins as I provide the 12v input through the fan connectors so it is much more useful for me to have them provide 3.3v out to the daughter board.

**Reply by skiphansen** [reply]
Thanks for the quick response Dave!  I've cloned your repo for a deeper look and to see if I can import it into Kcad.
